### PR TITLE
Remove CRuntime dependency

### PR DESCRIPTION
**Pull request by ddddxxx**

### CODE REVIEW

**Comment by wickwirew** [comment]
Unfortunately this will break the Linux build. It use to work this way but had some linking errors. Discussion here: https://github.com/wickwirew/Runtime/issues/18

**Comment by ddddxxx** [comment]
I see. Is there any reason to split CRuntime in separate repo?

**Comment by ddddxxx** [comment]
We can keep Runtime and CRuntime in one repo, like this.

**Comment by wickwirew** [comment]
tbh im not 100% sure. I was told that is how it has to work. I dont have a Linux machine so its difficult to test for me.

**Comment by ddddxxx** [comment]
I notice there is a `.travis.yml` file, but no CI applied to this pr. Can you make it work?

**Comment by wickwirew** [comment]
Fixed it

**Comment by MaxDesiatov** [comment]
Any updates on this PR? Could someone clarify if there's anything that prevents it from being merged?

**Comment by ddddxxx** [comment]
This pr is ready for review. @wickwirew

File: Runtime.podspec [hunk 1]
```diff
@@ -12,5 +12,4 @@ Pod::Spec.new do |s|
     s.platform     = :ios, "9.0"
     s.source       = { :git => "https://github.com/wickwirew/Runtime.git", :tag => s.version }
     s.source_files = 'Sources/Runtime/**/*.swift'
```
Comment (wickwirew) [inline Runtime.podspec]: Shouldnt this be `s.source_files = 'Sources/**/*.swift'` now since `CRuntime` is apart of the package?

**Comment by wickwirew** [comment]
Sorry for the late response! It looks good, but I think this will break CocoaPods at the moment. Once thats figured out Ill merge it

**Comment by MaxDesiatov** [comment]
This seems to be outdated after #66 is merged

**Comment by wickwirew** [comment]
Yea I meant to close this but it slipped my mind, Ill close it now.

**Pull request closed without merging**